src/readiness.ts

```diff
@@ -0,0 +1,56 @@
+import { Bee, BeeDebug, Utils } from '@ethersphere/bee-js'
+import { READINESS_TIMEOUT } from './config'
+import { logger } from './logger'
+import { StampsManager } from './stamps'
+
+export async function checkReadiness(bee: Bee, beeDebug: BeeDebug, stampManager?: StampsManager): Promise<boolean> {
```

Comment:
Could you please add test for this function? You can for example use the https://github.com/ffMathy/FluffySpoon.JavaScript.Testing.Faking (we have it already in devDependencies) to mock the classes and you can do simple unit tests with it.